### ISSUE TITLE
QskScrollView‘problem

### ISSUE THREAD

**Issue by hestrro** [uwerat/qskinny#485]
Uploading scroll.mp4…

**Comment by hestrro** [comment]
https://github.com/user-attachments/assets/bee14e29-0a5d-4179-876f-7bdc1d41d74b

**Comment by uwerat** [comment]
This seems to be a modified version of the thumbnails example ?

**Comment by hestrro** [comment]
Yes, you can test it in the thumbnails example. I encountered this situation when using QskScrollArea.

**Comment by hestrro** [comment]
I ran the demo program example directly without modifying it.
